scrc/utils/sql_select_utils.py

```diff
@@ -408,10 +411,6 @@ def get_region(canton):
         if np.isnan(canton):
             return None
         canton = Canton(canton)
-    if isinstance(canton, str):
-        for canton_enum in Canton:
-            if canton_enum.name == canton:
-                canton = canton_enum
```

Comment:
Is there a reason for removing this?

Reply:
No there is not, it was not removed on purpose. I will include it again.